app/containers/Applications/ApplicationDetailsCardItem.tsx

```diff
@@ -178,5 +183,14 @@ export default createFragmentContainer(ApplicationDetailsCardItemComponent, {
       ...FuelRowIdField_query
       ...EmissionCategoryRowIdField_query
     }
+  `,
+  applicationRevision: graphql`
+    fragment ApplicationDetailsCardItem_applicationRevision on ApplicationRevision {
+      naicsCode {
+        id
+        ...ProductRowIdField_naicsCode
+        ...ProductField_naicsCode
+      }
+    }
```

Comment:
I would retrieve this from the `formResult`, using `applicationRevisionByApplicationIdAndVersionNumber`